components/home/homepage-supported-apps.tsx

```diff
@@ -0,0 +1,52 @@
+import Image, { ImageLoaderProps } from "next/image";
+import { HomePageSection } from "./homepage-section";
+import supportedApps from "~/lib/content/officially-supported-apps";
+import { cloudinaryLoader } from "~/lib/image/cloudinary-loader";
+import { replaceCloudinaryPrefix } from "~/lib/image/cloudinary-utils";
+
+export function HomepageSupportedApps() {
+  return (
+    <HomePageSection className="px-4 py-6 space-y-4 text-center">
+      <h2 className="font-semibold">
+        Download aplikasi telemedicine yang
+        <br />
+        didukung Kemenkes:
+      </h2>
+      <ul className="grid grid-cols-2 gap-2 sm:gap-4 sm:grid-cols-4">
+        {supportedApps.application.map((app, index) => (
+          <li
+            key={index}
+            className="inline-flex flex-col items-center justify-center text-center relative  py-3 px-2"
```

Comment:
```suggestion
            className="inline-flex flex-col items-center justify-center text-center relative"
```

We can remove these padding to bring everything closer together.

Reply:
I just pushed new commit without padding. thanks